Library/Formula/agda.rb

```diff
@@ -10,13 +10,19 @@ class Agda < Formula
   sha256 "bc6def45e32498f51863d67acfbe048c039d630c6a36761ed27e99a5f68d7b27"
   revision 2
 
+  head "https://github.com/agda/agda.git", :branch => "master"
+
   bottle do
     sha256 "b5dfadfde1b45ca3d70f2ac812e98a6d72caf1d840d4cb9c4eee95e81d74ff0b" => :yosemite
     sha256 "4513aecea2fbf5d1a682e04e4cccfd795d48836bcf1b16f79c24fa3bd6482854" => :mavericks
     sha256 "d0e17808ee896cdedc499b010b67961dc3db3c430c501faef16c4b4482238447" => :mountain_lion
   end
 
-  head "https://github.com/agda/agda.git", :branch => "master"
+  head do
+    resource "stdlib" do
```

Comment:
The head `url` should be placed in this block as well, to keep everything in one place.